lib/src/model.dart

```diff
@@ -1326,6 +1350,28 @@ class Extension extends Container
         ElementType.from(_extension.extendedType, library, packageGraph);
   }
 
+  /// Returns [true] if this extension could be applicable to any possible
+  /// instantiation of [c].
```

Comment:
This does not sound mathematically rigorous :-)
Maybe "...if there is an instantiation of [c] to which this extension could be applied".

Reply:
yes, better.  Done.